src/components/ErrorBoundary.js

```diff
@@ -1,31 +1,57 @@
 import React from "react";
 import Alert from "@material-ui/lab/Alert";
 
-function defaultFormatter(error) {
+/**
+ * A basic FallbackComponent for the {@link ErrorBoundary}. Displays a "yikes"
+ * message using a Material UI Alert.
+ */
+export function BasicError(props) {
   return (
     <Alert severity="error">
       Yikes! Something went wrong loading the OSCAL data. Sorry, we&apos;ll look
-      into it. ({error.message})
+      into it. ({props.error.message})
```

Comment:
While we're here, can we remove the "Sorry, we'll look into it." sentence please?